### DISCUSSION TITLE
Qiitaの記事に遷移しようとすると「Not connected インターネットにつながっていません。」が出てくる

### DISCUSSION THREAD

**Discussion by oyaken0717** [increments/qiita-discussions#374] [ヘルプ] [answered]
なぜでしょうか？

**Comment by tanaka-asami415** [comment, marked as answer]
@oyaken0717 

ご不便をおかけして申し訳ございません。
まずは下記をお試しいただけますでしょうか。

- キャッシュの削除

こちらで解決しない場合は、下記の情報を[サポートフォーム](https://support.qiita.com/hc/ja/requests/new)へお送りいただけますと幸いです。
（サポートフォームも閲覧できない場合は、support@qiita.com へご連絡ください）

1. 現象発生日時
2. 事象が発生するOS・ブラウザ名、及び各バージョン
3. 事象が発生する端末より異なるブラウザを利用した場合に事象が再現するかの確認結果
4. ご利用のプロバイダサービス名

お手数をおかけいたしますが、どうぞよろしくお願いいたします。

**Reply by oyaken0717** [reply]
サポートフォームに送りましたが
やっぱり解消方法はわからない感じですかね？

**Reply by tomohiro-n** [reply]
同様の現象に遭遇したので共有します。
- Brave [Version 1.44.105 Chromium: 106.0.5249.91 (Official Build) (x86_64)](https://brave.com/latest/): "Not connected" が出てくる
  - その後最新版( [Version 1.46.144 Chromium: 108.0.5359.128 (Official Build) (x86_64)](https://brave.com/latest/) )にしても同じでした
- 同ブラウザ Private Window: "NET::ERR_CERT_AUTHORITY_INVALID" エラー
- 普段使わないChrome Version 108.0.5359.124 (Official Build) (x86_64): 正常に表示される(証明書異常なしと判断される)

という感じです。これ以上僕からは深追いしませんが、参考になれば幸いです。

<img width="1359" alt="image" src="https://user-images.githubusercontent.com/4282112/208235128-dffd68e7-03d7-437c-9328-8391415be890.png">
左: BraveのPrivate Window
右: Chrome

**Reply by tanaka-asami415** [reply]
@oyaken0717

> サポートフォームに送りましたが
やっぱり解消方法はわからない感じですかね？

ご対応いただきありがとうございます🙇
サポートフォームへ届いたお問合せにつきましては、全て返信を行っているのですが
サポートからの返信がない状況でしょうか？

その場合は、何らかの原因でoyaken0717 様のメールが届いていない可能性がございますので
差支えないようでしたら、Qiitaのアカウント名を教えていただけないでしょうか。
こちらでお調べして、Qiitaへご登録のメールアドレスへご連絡させていただきたく存じます🙇

@tomohiro-n
詳細を共有いただきありがとうございます🙏

**Reply by oyaken0717** [reply]
@tanaka-asami415 
返信が遅れてしまい大変申し訳ございませんでした。
別件でOSのアップデートを何がなんでもやらねばならずそちらに時間を費やしてしまいました。
OSのアップデートをしたところ問題なく表示されるようになりましたので
問題が解決しました。
ありがとうございました。

**Reply by tanaka-asami415** [reply]
@oyaken0717 
とんでもないです！問題が解決されたようで何よりです🙌✨

**Comment by scivola** [comment]
同じ現象が発生して，キャッシュをクリアしたところ https://github.com/increments/qiita-discussions/discussions/418 の状態になり，全く使えません。

**Reply by tanaka-asami415** [reply]
@scivola 
まだお試しでなければ、一度再起動もお試しいただけますと幸いです。
それでも解決しない場合は、下記の情報を[サポートフォーム](https://support.qiita.com/hc/ja/requests/new)へお送りいただけますと幸いです。
（サポートフォームも閲覧できない場合は、support@qiita.com へご連絡ください）

- IPアドレス
- ディベロッパーツールのConsoleにエラーが出ている場合はエラーのキャプチャ
- 環境（OS、ブラウザ、OSとブラウザのバージョン）

どうぞよろしくお願いいたします。

**Reply by scivola** [reply]
すみません，「再起動」というのは何の再起動でしょうか。

なお，https://github.com/increments/qiita-discussions/discussions/418 は Firefox が証明書を信用してない，という問題と思われます。

**Reply by tanaka-asami415** [reply]
@scivola 
失礼いたしました🙏ブラウザとデバイスの再起動をお試しでなければ、一度お試しください。
それでも解決されない場合は、サポートフォームへ以下をご連絡いただけますと幸いです。

- IPアドレス
- ディベロッパーツールのConsoleにエラーが出ている場合はエラーのキャプチャ
- 環境（OS、OSのバージョン）

https://github.com/increments/qiita-discussions/discussions/418 詳細を追記いただきありがとうございます🙏

**Reply by scivola** [reply]
ご返事が遅くなってすみません。
ブラウザーや OS の再起動は何度も試しました。

サポートフォームにレポート送らなくちゃ，と思いつつ，時間が取れず先延ばしにしていたところ，今日（2023/01/28）の夕方（？）になって普通に閲覧できるようになりました。
問題が一時的に再現してないだけなのか根本的に解決したのか分かりませんが，何か対処されましたでしょうか？

こちら側の環境で，今朝と夕方とで違っていることと言えば，一つだけ思い当たることがあります。
macOS Big Sur のアップデートを行いました（現在のバージョンは 11.7.3）。
これが関係しているとすると，OS とブラウザーとサイトの組み合わせで何か問題が生じていたのでしょうかね。
それにしても不可解な。

**Reply by tanaka-asami415** [reply]
@scivola 
ご報告いただきありがとうございます🙏

> 何か対処されましたでしょうか？

Qiita側では特に何もしておらずでして…
このDiscussionを立ててくださった oyaken0717さんも、OSのアップデート後に閲覧ができるようになったようなので
もしかすると何か関係があったのかもしれません…

具体的な原因は分かっておらず、あまりお役に立てず申し訳ございません。
ひとまず問題が解決されたようで何よりです🙇
一旦 #418 もクローズさせていただきます🙇

**Reply by scivola** [reply]
了解です。
では OS アップデートで解決した可能性が高そうですね。
